flocker/common/test/test_version.py

```diff
@@ -182,6 +180,7 @@ def test_is_pre_release(self):
                 version_case.is_pre_release,
             )
 
+        @skipUnless(PACKAGING_INSTALLED, "``packaing`` not installed.")
```

Comment:
:grey_question: (optional as it's not your change) s/packaing/packaging/g (also below)